packages/terra-demographics-banner/src/DemographicsBanner.jsx

```diff
@@ -83,6 +83,7 @@ const DemographicsBanner = ({
   return (
     <DemographicsBannerDisplay
       {...customProps}
+      className="demographics-banner"
```

Comment:
"demographics-banner" is unbound, we'll need to locally bind it to prevent global style collisions.

Reply:
classname is not required here, [removed](https://github.com/cerner/terra-core/pull/2856/files#diff-c554f9bb4d56617bb2a5b86d37b215f2)